lms/envs/common.py

```diff
@@ -3902,6 +3902,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
         "secondary_email_enabled",
         "year_of_birth",
         "phone_number",
+        "activation_key",
```

Comment:
Question: Which serializer is using this dataset?

Reply:
https://github.com/edx/edx-platform/blob/01baa9ab9c621e2cd1a28c746e276ebe05820935/openedx/core/djangoapps/user_api/accounts/serializers.py#L93

Please see this link. It uses configuration and return only allowed keys.